stations-along-route/queries.js

```diff
@@ -0,0 +1,91 @@
+import qql from 'graphql-tag';
+
+/*
+ * In this example we request a route from Amsterdam, Netherlands to Utrecht, Netherlands
+ * The changing conditions are:
+ *   - full battery at Amsterdam, Netherlands
+ *   - no desired range at Utrecht, Netherlands
+ *   - EV can charge at CHadMO changers
+ *   - should use climate (temperature and weather conditions)
+ *   - the EV driver can drive 40 km  less than the EV specs (specs is 440 km, custom range is 400 km)
+ *   - min power of chargers is 43 kWh
+ *   - one passenger in the car (drive alone)
+ *   - stations along the route radius is 2km
+ */
+export const createRoute = qql`
+mutation newRoute{
+    newRoute(
+      input: {
+        ev: {
+          id: "5d161be5c9eef46132d9d20a"
+          battery: {
+            capacity: { value: 72.5, type: kwh }
+            stateOfCharge: { value: 50.5, type: kwh }
+            finalStateOfCharge: { value: 0, type: kwh }
+          }
+          plugs: { chargingPower: 150, standard: TESLA_S }
+          adapters: [
+            { chargingPower: 150, standard: IEC_62196_T2_COMBO }
+            { chargingPower: 150, standard: CHADEMO }
+          ]
+          climate: true
+          minPower: 43
+          numberOfPassengers: 1
+        }
+        routeRequest: {
+          origin: {
+            type: Feature
+            geometry: { type: Point, coordinates: [4.8951679, 52.3702157] }
+            properties: { name: "Amsterdam, Netherlands" }
+
+          }
+          destination: {
+            type: Feature
+            geometry: { type: Point, coordinates: [8.0472, 52.2799] }
+            properties: { name: "Osnabrück, Germany" }
+          }
+          stationsAlongRouteRadius: 2000
```

Comment:
We currently hardcode this value. Did you think about making this a dynamic variable? It can be between 500 and 5000 meters. Maybe it would be nice to show this capability or at the very least explain its.

Reply:
We did consider this. But the difference is really minimal. So we decided against it. I can maybe add extra comments in the code to further explain it or refer to our documentation.